src/IndustrialRobot/IndustrialRobot/ViewModels/MainViewModel.cs

```diff
@@ -1,14 +1,97 @@
 ﻿using Caliburn.Micro;
 using System;
 using System.Collections.Generic;
+using System.IO.Ports;
 using System.Linq;
 using System.Text;
 using System.Threading.Tasks;
+using System.Windows;
+using System.Windows.Controls;
+using System.Windows.Data;
+using System.Windows.Documents;
+using System.Windows.Input;
+using System.Windows.Media;
+using System.Windows.Media.Imaging;
+using System.Windows.Shapes;
+using IndustrialRobot.Views;
+using GalaSoft.MvvmLight.Command;
+using IndustrialRobot.Views.Interfaces;
 
 namespace IndustrialRobot.ViewModels
 {
-    public class MainViewModel : Screen
+    public class MainViewModel : Window 
```

Comment:
Czy to jest "Window" z WPFa? jesli tak to tak nie powinno byc. ViewModel, powinien byc zupelnie niezalezny od warstwy prezentacji, tak aby np w razie potrzeby latwo bylo zamienic warstwe prezentacji na jakas inna, np WEBowa. A tak jak ViewModel ma zaleznosc do WPFa no to do weba go juz nie wrzucisz....

Reply:
done